accord-core/src/main/java/accord/local/Command.java

```diff
@@ -351,13 +369,14 @@ public enum CommitOutcome { Success, Redundant, Insufficient }
     // relies on mutual exclusion for each key
     public CommitOutcome commit(SafeCommandStore safeStore, AbstractRoute route, @Nullable RoutingKey progressKey, @Nullable PartialTxn partialTxn, Timestamp executeAt, PartialDeps partialDeps)
     {
-        if (hasBeen(Committed))
+        if (hasBeen(PreCommitted))
```

Comment:
What is going on here with Precommitted vs Committed?

Reply:
If we've already been `PreCommitted` our `executeAt` should already be correct, so we just check it.